src/components/card/card.module.css

```diff
@@ -48,6 +50,85 @@
 .channelName {
     padding-top: 5px;
     margin-top: auto;
-    font-weight:lighter;
+    font-weight: lighter;
     color: rgb(171, 171, 171)
+}
+
+.historyVideoCard {
+    width: 80rem;
+    height: 15rem;
+    border-radius: 8px;
+    background-color: var(--COLOR-LIGHT-BACKGROUND);
+    margin-bottom: 2rem;
+    cursor: pointer;
+    display: flex;
+}
+
+.historyCardChannelName {
+    color: var(--COLOR-GREY-200);
+}
+
+.historyCardImageContainer img {
+    width: 20rem;
+    height: 100%;
+    object-fit: cover;
+    border-radius: 8px 0px 0px 8px;
```

Comment:
`0px` ?? `0` is better ?

Reply:
Yups there should not be any unit with 0. Made the changes.